performance/benchmark.rb

```diff
@@ -5,13 +5,14 @@
 profiler = ThemeRunner.new
 
 Benchmark.ips do |x|
-  x.time = 60
+  x.time = 10
```

Comment:
the slowest benchmark iterates 150 times in 10 seconds so I thought that was representative, I can change it back if you want